ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/table-viz.hbs

```diff
@@ -19,20 +19,16 @@
 <div class="table-rearrange">
 </div>
 <div class="table-options">
-        <button class="options-rearrange is-button">
-            Rearrange Columns
-            <span class="fa fa-columns"></span>
-        </button>
-        <button class="options-visibility is-button">
-            Hide / Show Columns
-            <span class="fa fa-eye"></span>
-        </button>
-        <button class="options-export-all is-button">Export all to CSV
-            <span class="fa fa-share"></span>
-        </button>
-        <button class="options-export-visible is-button">Export visible to CSV
-            <span class="fa fa-share"></span>
-        </button>
+    <button class="options-rearrange is-button">
+        Rearrange Columns
+        <span class="fa fa-columns"></span>
+    </button>
+    <button class="options-visibility is-button">
+        Hide / Show Columns
```

Comment:
❓ Would `Hide/Show Columns` be clearer, or do we prefer the spaces around the `/`?

Reply:
I don't have a strong opinion either way so I'm fine with changing it